actors/verifreg/tests/verifreg_actor_test.rs

```diff
@@ -0,0 +1,240 @@
+#![deny(unused_must_use)] // Force unwrapping Result<_, Err>
+
+use fvm_shared::{HAMT_BIT_WIDTH, MethodNum};
+use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser::BigIntDe;
+use fvm_shared::encoding::RawBytes;
+use lazy_static::lazy_static;
+
+use fil_actor_verifreg::{Actor as VerifregActor, AddVerifierClientParams, AddVerifierParams, DataCap, Method, MINIMUM_VERIFIED_DEAL_SIZE, State};
+use fil_actors_runtime::{ActorError, make_empty_map, make_map_with_root_and_bitwidth, SYSTEM_ACTOR_ADDR};
+use fil_actors_runtime::test_utils::*;
+
+lazy_static! {
+    static ref ROOT_ADDR: Address = Address::new_id(101);
+    static ref VERIFIER_ALLOWANCE: DataCap = MINIMUM_VERIFIED_DEAL_SIZE.clone() + DataCap::from(42);
+}
+
+fn construct_runtime() -> MockRuntime {
```

Comment:
I would put that in a separate util module so that it is reusable by other test modules and then make each test modulr a flat file.

Reply:
I'm only intending on one test file for now. I'll move it if I split into multiple.